library/src/test/java/bio/terra/landingzone/stairway/flight/create/CreateAzureLandingZoneDbRecordStepTest.java

```diff
@@ -0,0 +1,118 @@
+        LandingZoneRecord.builder()
+            .landingZoneId(UUID.randomUUID())
+            .description("Definition:%s Version:%s")
+            .displayName("test")
+            .definition("test")
+            .properties(Map.of())
+            .resourceGroupId(UUID.randomUUID().toString())
+            .tenantId(UUID.randomUUID().toString())
+            .subscriptionId(UUID.randomUUID().toString())
+            .region("test-region")
+            .billingProfileId(UUID.randomUUID())
+            .build();
+
+    var landingZoneRequest =
+        LandingZoneRequest.builder()
+            .definition(expectedLandingZone.definition())
+            .version(expectedLandingZone.version())
+            .billingProfileId(expectedLandingZone.billingProfileId())
+            .parameters(expectedLandingZone.properties())
+            .build();
+    var billingProfile =
+        new ProfileModel()
+            .id(expectedLandingZone.billingProfileId())
+            .managedResourceGroupId(expectedLandingZone.resourceGroupId())
+            .tenantId(UUID.fromString(expectedLandingZone.tenantId()))
+            .subscriptionId(UUID.fromString(expectedLandingZone.subscriptionId()));
+    var inputMap =
+        FlightTestUtils.prepareFlightInputParameters(
+            Map.of(
+                LandingZoneFlightMapKeys.LANDING_ZONE_CREATE_PARAMS, landingZoneRequest,
+                LandingZoneFlightMapKeys.LANDING_ZONE_ID, expectedLandingZone.landingZoneId(),
+                LandingZoneFlightMapKeys.BILLING_PROFILE, billingProfile));
+    var workingMap =
+        FlightTestUtils.prepareFlightWorkingParameters(
+            Map.of(
+                GetManagedResourceGroupInfo.TARGET_MRG_KEY,
+                new TargetManagedResourceGroup(
+                    expectedLandingZone.resourceGroupId(), expectedLandingZone.region())));
+    when(context.getInputParameters()).thenReturn(inputMap);
+    when(context.getWorkingMap()).thenReturn(workingMap);
+
+    when(landingZoneDao.createLandingZone(any()))
+        .thenAnswer(
+            invocation -> {
+              var landingZone = invocation.getArgument(0, LandingZoneRecord.class);
+              assertThat(landingZone.displayName(), equalTo(expectedLandingZone.displayName()));
+              assertThat(landingZone.properties(), equalTo(expectedLandingZone.properties()));
+              assertThat(
+                  landingZone.resourceGroupId(), equalTo(expectedLandingZone.resourceGroupId()));
+              assertThat(landingZone.tenantId(), equalTo(expectedLandingZone.tenantId()));
+              assertThat(
+                  landingZone.subscriptionId(), equalTo(expectedLandingZone.subscriptionId()));
+              assertThat(
+                  landingZone.billingProfileId(), equalTo(expectedLandingZone.billingProfileId()));
+              assertThat(landingZone.region(), equalTo(expectedLandingZone.region()));
+              return expectedLandingZone.landingZoneId();
```

Comment:
Should we return actual value for landing zone. Like landingZone.landingZoneId() instead?

Reply:
Good point. 
It shouldn't affect anything, in practice - but that is the correct behavior.